.gitignore

```diff
@@ -20,5 +20,12 @@ pnpm-debug.log*
 # macOS-specific files
 .DS_Store
 
-# jetbrains setting folder
+# IDE
 .idea/
+.vscode/
+
+# Hugo
+.hugo/
```

Comment:
The website is built with Astro, not Hugo. I think all these files if they are deleted won't show up again.

Reply:
Don't have any negative impact to have them in the .gitignore though...

Reply:
thx, i wasn't sure why my repo was dirty, so cleaned it all up.  I guess its ok to have extras - not a big deal